lib/order.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'menu'
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :order, :total, :menu
+
+  def initialize
+    @menu = Menu.new
+    # @menu.print_menu
+    @order = []
+  end
+
+  def add_items(item)
+    @order << @menu.menu[item - 1]
+  end
```

Comment:
Clear way to add items with numbers - but what if they put the name instead of a number?